util/ClientEndpoint.php

```diff
@@ -48,6 +48,12 @@ public function renderClass(): string
         $class = file_get_contents(self::CLIENT_CLASS_TEMPLATE);
         // use Namespace
         $useNamespace = '';
+        
+        // The following namespaces do not have OpenSearch API specifications
```

Comment:
Ideally this kind of stuff should be data-driven, e.g. with a folder structure `ssl/something.yaml`.